src/common.h

```diff
@@ -5,6 +5,18 @@
 #include <config.h>
 #endif
 
+#ifdef has_jemalloc
+#include <jemalloc/jemalloc.h>
+#undef HAVE_C_MALLOPT
+#undef HAVE_C_MALLINFO
```

Comment:
please do not do this, you can never know what implications are.
if you want define something to skip definition, but most probably you do not want to define them at first in configure.